[DEV-980] Adds external rewards redemption documentation (#5)
KudosInc/slate: source/includes/_redemption_history.md

## Redemptions History

<a id="opIdredemptionHistory"></a>

> This end points fetches redemption history for a user or the complete organization.

Reviewer: ```suggestion
This end points fetches the redemption history scoped by a user or by the organization.
```

```shell
$ curl -i -X GET -H 'Content-Type: application/json' --url 'https://api-client.kudosnow.com/v1/redemptions?apikey=your_key_here&userId=1&page=1&perPage=15'
```
```javascript
var request = require('request');

var options = {
  uri: 'https://api-client.kudosnow.com/v1/redemptions?userId=1&page=1&perPage=15',
  method: 'GET',
  headers: {
    "apikey": "your_key_here"
  }
};

request(options, function (error, response, body) {
  ...
});
```

### HTTP Request 
`GET /redemptions` 

**Parameters**

|Name|Located in|Description|Required|Type|
|---|---|---|---|--|

> |userId|query|User Id|false|integer|

Reviewer: ```suggestion
|userId|query|User ID|false|integer|
```

|page|query|Page number|false|integer|

> |perPage|query|Count of items per page (15 by default)|false|integer|

Reviewer: ```suggestion
|perPage|query|Number of redemptions per page. Defaults to 15|false|integer|
```
